src/App.tsx

```diff
@@ -29,127 +81,100 @@ export const App: React.FC = () => {
+                completed: todo?.completed,
+              })}
+            >
+              <label className="todo__status-label">
+                <input
+                  type="checkbox"
+                  className="todo__status"
+                  defaultChecked={todo?.completed}
+                />
+              </label>
+
+              <span className="todo__title">{todo?.title}</span>
+
+              <button type="button" className="todo__remove">
+                ×
+              </button>
+
+              <div className="modal overlay">
+                <div className="modal-background has-background-white-ter" />
+                <div className="loader" />
+              </div>
             </div>
-          </div>
+          ))}
         </section>
 
-        {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer">
-          <span className="todo-count">
-            3 items left
-          </span>
-
-          {/* Active filter should have a 'selected' class */}
-          <nav className="filter">
-            <a href="#/" className="filter__link selected">
-              All
-            </a>
-
-            <a href="#/active" className="filter__link">
-              Active
-            </a>
-
-            <a href="#/completed" className="filter__link">
-              Completed
-            </a>
-          </nav>
-
-          {/* don't show this button if there are no completed todos */}
-          <button type="button" className="todoapp__clear-completed">
-            Clear completed
-          </button>
-        </footer>
+        {todos?.length > 0 && (
+          <footer className="todoapp__footer">
+            <span className="todo-count">{`${uncompletedTodos?.length} items left`}</span>
+
+            <nav className="filter">
+              <a
+                href="#/"
+                className={cn('filter__link', {
+                  selected: filterStatus === Status.ALL,
+                })}
+                onClick={() => setFilterStatus(Status.ALL)}
+              >
+                All
+              </a>
+
+              <a
+                href="#/active"
+                className={cn('filter__link', {
+                  selected: filterStatus === Status.ACTIVE,
+                })}
+                onClick={() => setFilterStatus(Status.ACTIVE)}
+              >
+                Active
+              </a>
+
+              <a
+                href="#/completed"
+                className={cn('filter__link', {
+                  selected: filterStatus === Status.COMPLETED,
+                })}
+                onClick={() => setFilterStatus(Status.COMPLETED)}
+              >
+                Completed
+              </a>
+            </nav>
+
+            <button
+              type="button"
+              className="todoapp__clear-completed"
+              style={{ opacity: completedTodos.length > 0 ? '1' : '0' }}
+              disabled={!completedTodos}
```

Comment:
You are using '!completedTodos' for disabling the button. It should be '!completedTodos?.length' as completedTodos is an array and if it's empty, the button should be disabled.